### DISCUSSION TITLE
Is it possible to trigger a build automatically after text edit on SSR page?

### DISCUSSION THREAD

**Discussion by SarahVanDenBerghe** [vercel/next.js#21692] [Help]
Hi, I'm new to NextJS but I'm working on a project for my university and would love some input! I'm in a situation where I server-side render a list of user project posts (which in itself is a page). I did want to SSR that to improve SEO and reduce calls to the Firebase backend. However, the user would be able to edit this post. So technically, this would require their page to be rebuild. 

I was wondering how that was done, I've searched some things online and came across Cron Job, where a rebuild could be scheduled by time. But I'm looking for something that I can use that once the user edits his project (through an edit form), their page would be rebuild. So not necessarily a timed build schedule. Projects wouldn't be edited as often hence why I'd prefer it if it were still SSR but just build again after an edit. Is this possible?

**Comment by michrome** [comment]
Hey @SarahVanDenBerghe if things aren't edited often, you could use a [Deploy Hook](https://vercel.com/docs/more/deploy-hooks) when a change is made.
